### DISCUSSION TITLE
RelationManager label translation

### DISCUSSION THREAD

**Discussion by ALSHIHAB** [filamentphp/filament#8606] [Help] [answered]
How to translate RelationManager relationship label. I've used:      protected static ?string $title = 'My documents';
But couldn't translate it.

**Comment by leandrocfe** [comment, marked as answer]
You can change via `$table`

YourRelationManager.php
```php
$table
    ->heading('My documents')
    ->pluralModelLabel('My documents')
    ->modelLabel('My Document')
    ...
```

**Reply by ALSHIHAB** [reply]
Perfect. Thanks @leandrocfe
